tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -319,6 +325,84 @@ void test_area(const Solution& solution, const Strahlkorper<Fr>& strahlkorper,
   CHECK(irreducible_mass == approx(expected_irreducible_mass));
 }
 
+// Let P = surface_integral_of_scalar(J^i \tilde{s}_i \sqrt(-g))
+// where \tilde{s}_i is the spatial unit one-form to the Strahlkorper,
+// normalized with the flat metric \tilde{s}_i \tilde{s}_j \delta^{ij} = 1,
+// and where euclidean_area_element is passed into surface_integral_of_scalar.
+//
+// Let Q = surface_integral_of_scalar(J^i s_i \alpha), where \alpha is the
+// lapse, where s_i is the spatial unit one-form to the Strahlkorper,
+// normalized with the 3 metric s_i s_j \gamma^{ij} = 1, and where
+// the area_element computed with $\gamma_{ij}$ is passed into
+// surface_integral_of_scalar.
+//
+// This tests that P=Q for an arbitrary 3-vector J^i.
```

Comment:
Very minor, feel free to ignore: the comment uses P vs. Q, but the code uses 1 vs. 2. Can these be made consistent? For example, by changing P to "I1" and Q to "I2", or whatever else you prefer.